spec/models/manageiq/providers/vmware/infra_manager/vm/reconfigure_spec.rb

```diff
@@ -5,7 +5,7 @@
       :name            => 'test_vm',
       :raw_power_state => 'poweredOff',
       :storage         => FactoryGirl.create(:storage, :name => 'storage'),
-      :hardware        => FactoryGirl.create(:hardware, :cpu2x2, :ram1GB, :virtual_hw_version => "07")
+      :hardware        => FactoryGirl.create(:hardware, :cpu4x2, :ram1GB, :virtual_hw_version => "07")
```

Comment:
That was so I didn't have to override the default options when I wanted to test just memory or just cpu, I just set it to match https://github.com/agrare/manageiq/blob/a908c662e367e695a58e365550bce7f769b0058f/spec/models/manageiq/providers/vmware/infra_manager/vm/reconfigure_spec.rb#L65

If you look at the rest of the changes in that commit I was able to remove a bunch of code that set the options to the current value, for example https://github.com/ManageIQ/manageiq/pull/12275/commits/a908c662e367e695a58e365550bce7f769b0058f#diff-fafec56c2822f5c78227a8a11313bc6fL124